### PR TITLE
Expanded info

### PR DESCRIPTION
**Pull request by kieran-whittenburg**

### CODE REVIEW

File: whois/parser.py [hunk 1]
```diff
@@ -272,26 +278,44 @@ def load(domain, text):
...
+        'emails': EMAIL_REGEX,   # list of email s
+        'address': 'address: *(.+)'
+    }
+
     def __init__(self, domain, text):
         if 'No match for "' in text:
             raise DomainNotFoundError(text)
         else:
-            WhoisEntry.__init__(self, domain, text)
+            WhoisEntry.__init__(self, domain, text, self.regex)
```
Comment (morrissimo) [inline whois/parser.py:312]: Thoughts on this (and ditto the same thing further down in `WhoisIANA`):
1. use `super()`: `super(WhoisSpace, self).__init__(domain, text, self.regex)`
2. drop the `else:` - the if clause blows up execution anyway, so you can lose the `else:` and save a little bit on overall complexity

It should work fine as written, but using `super()` is the correct way to call a superclass method; dropping the `else` is just a stylistic issue, up to you

**Review by morrissimo — COMMENTED** [review]
looks as good as this module can look lmao - left some feedback on some optional changes, so I'll wait on merging/etc until you respond

File: whois/parser.py (hunk 1 above)
Comment (kieran-whittenburg) [reply whois/parser.py:312]: That's actually an artifact of the original design (aka not my design), but I can update it if you want. For every class. Uhhh.

Comment (morrissimo) [reply whois/parser.py:312]: Then file under "hellllll nope" - let's leave it uniformly shitty :)

**Pull request merged**